tests/server/test_resource_list.py

```diff
@@ -406,3 +406,60 @@ async def test_paging_param_validation(server, client, env_with_resources):
 
     result = await client.resource_list(env_with_resources.id, limit=2, first_id="1234", last_id="5678")
     assert result.code == 400
+
+
+@pytest.mark.asyncio
+async def test_deploy_summary(server, client, env_with_resources):
+    """ Test querying the deployment summary of resources."""
+    env = env_with_resources
+    expected_summary = {
+        "total": 6,
+        "by_state": {
+            "unavailable": 1,
+            "skipped": 1,
+            "deployed": 2,
+            "deploying": 1,
+            "available": 1,
+            "dry": 0,
```

Comment:
this state should never be in here I think.

Reply:
Only the `dry` state? The rest of them are ok?